src/pages/notes/little-dorrit/mirador.astro

```diff
@@ -0,0 +1,11 @@
+---
+import MiradorFullScreenLayout from "../../../layouts/mirador-full-screen-layout.astro";
+import Mirador from "../../../components/Mirador";
+
+const manifest =
+  "https://dickensnotes.github.io/dickens-annotations/img/derivatives/iiif/littledorrittranscription/manifest.json";
+---
+
+<MiradorFullScreenLayout title="Little Dorrit Mirador viewer" description="TODO">
```

Comment:
Need to fill in this TODO in the meta description

Reply:
added description in metadata